code/game/machinery/doors/airlock.dm

```diff
@@ -312,13 +319,31 @@ About the new airlock wires panel:
 			if(p_open)
 				panel_overlay = get_airlock_overlay("panel_opening", overlays_file)
 
-	overlays.Cut()
-	overlays += frame_overlay
-	overlays += filling_overlay
-	overlays += lights_overlay
-	overlays += panel_overlay
-	overlays += weld_overlay
-	overlays += sparks_overlay
+	//doesn't use overlays.Cut() for performance reasons
```

Comment:
You're certain that the below is actually faster/more efficient than using Cut()?